mesheryctl/internal/cli/root/system/fixtures/.meshery/manifests/default.yaml

```diff
@@ -0,0 +1,439 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.3.0
+  name: brokers.meshery.layer5.io
```

Comment:
whoaaa!! damnn,My bad @piyushsingariya . I was playing around with something,and somehow copied these files here. And somehow didn't even notice before pushing the commit. They are being used nowhere. They are here because of my ignorance. Will remove them in next commit.